bin/internal/update_dart_sdk.ps1

```diff
@@ -38,7 +38,6 @@ if ((Test-Path $engineStamp) -and ($engineVersion -eq (Get-Content $engineStamp)
     return
 }
 
-Write-Host "Downloading Dart SDK from Flutter engine $engineVersion..."
```

Comment:
I removed these messages because redirecting pwsh stdout to stderr applies the exception formatting that cannot be removed

Reply:
Not sure what you mean. What is the exception formatting, and why is it not desirable? Because it looks like a failure?

Reply:
For example: https://i.stack.imgur.com/Lg7aA.png